### PR TITLE
Add module supporting AWS VPC Peering Connections - follow up

### PR DESCRIPTION
**Pull request by tendant**
This is the follow up pull request originated from #367

### CODE REVIEW

File: cloud/amazon/ec2_vpc_peering.py [hunk 1]
```diff
@@ -0,0 +1,611 @@
...
+
+DOCUMENTATION = '''
+---
+module: ec2_vpc_peering
+short_description: manage AWS VPC Peering Connections
+description:
+    - Create, list or reject/delete AWS VPC peering connections.
+    - This module has a dependency on python-boto
+version_added: "2.0"
+author: Herby Gillot <herby.gillot@gmail.com>
```
Comment (gregdek) [inline cloud/amazon/ec2_vpc_peering.py]: Let's make sure that both your Github ID and Herby's Github ID are included here.

**Comment by gregdek** [comment]
Thanks @tendant and @herbygillot -- I know this one has gone through the wringer.

One more change I see: please add Github IDs as requested, and then we'll try to get this one reviewed and merged asap. Thanks for your patience here.

(Please make sure to add a comment with the text "ready_for_review" when you're done so it hits by inbox, and we will continue with the review process asap.  Thanks!)

**Comment by tendant** [comment]
ready_for_review

**Comment by andynelson** [comment]
The state choices here seem a little like overkill, and tricky to use in practice.

I need to create peering relationships between VPCs across accounts fairly often, and would find this easier to use with a simple 'present' pairing that will create or accept a peering request that matches the given criteria... (the return dictionary can be examined to see whether this is a peering that is active, or still needs to be accepted)

If I have an inventory for each AWS account, and define the peering relationship in each one, it feels more declarative to be able to run the same play against both inventories, in either order with an active peering relationship as the result.

**Comment by gregdek** [comment]
I'm going to interpret @andynelson remarks as needs_revision.

@tendant Please make the suggested revisions, and when you're done, please add a comment with the text "ready_for_review" and we will continue with the review process.  Thanks!

**Comment by gregdek** [comment]
@tendant A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes.

**Comment by tendant** [comment]
@gregdek I will take a look this during the weekend.

**Comment by gregdek** [comment]
@tendant Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If we don't hear from you within another 14 days, we will close this pull request.

**Comment by gregdek** [comment]
@tendant A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@tendant Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@tendant A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@tendant Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@tendant A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@tendant Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@tendant A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@tendant Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@tendant A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by andynelson** [comment]
Has the need for this been alleviated, since there's a module in ansible-modules-extras that handles vpc peering already? https://github.com/ansible/ansible-modules-extras/blob/devel/cloud/amazon/ec2_vpc_peer.py

**Comment by gregdek** [comment]
@tendant Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@tendant A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@tendant Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by tendant** [comment]
Close this pull request, since this functionality is already available in ansible 2.

**Pull request closed without merging**